addons/website_blog/security/ir.model.access.csv

```diff
@@ -2,16 +2,22 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 blog_blog_public,blog.blog,model_blog_blog,base.group_public,1,0,0,0
 blog_blog_portal,blog.blog,model_blog_blog,base.group_portal,1,0,0,0
 blog_blog_employee,blog.blog,model_blog_blog,base.group_user,1,0,0,0
-blog_blog,blog.blog,model_blog_blog,website.group_website_designer,1,1,1,1
+blog_blog,blog.blog,model_blog_blog,website.group_website_designer,1,0,0,0
+blog_blog_manager,blog.blog,model_blog_blog,website_blog.group_website_blog_manager,1,1,1,1
 blog_post_public,blog.post,model_blog_post,base.group_public,1,0,0,0
 blog_post_portal,blog.post,model_blog_post,base.group_portal,1,0,0,0
 blog_post_employee,blog.post,model_blog_post,base.group_user,1,0,0,0
-blog_post,blog.post,model_blog_post,website.group_website_designer,1,1,1,1
+blog_post,blog.post,model_blog_post,website.group_website_designer,1,0,0,0
+blog_post_manager,blog.post,model_blog_post,website_blog.group_website_blog_manager,1,1,1,1
 blog_tag_public,blog.tag,model_blog_tag,base.group_public,1,0,0,0
 blog_tag_portal,blog.tag,model_blog_tag,base.group_portal,1,0,0,0
 blog_tag_employee,blog.tag,model_blog_tag,base.group_user,1,0,0,0
-blog_tag_edition,blog.tag,model_blog_tag,website.group_website_designer,1,1,1,1
+blog_tag_edition,blog.tag,model_blog_tag,website.group_website_designer,1,0,0,0
+blog_tag_manager,blog.tag,model_blog_tag,website_blog.group_website_blog_manager,1,1,1,1
 blog_tag_category_public,blog.tag.category,model_blog_tag_category,base.group_public,1,0,0,0
 blog_tag_category_portal,blog.tag.category,model_blog_tag_category,base.group_portal,1,0,0,0
 blog_tag_category_employee,blog.tag.category,model_blog_tag_category,base.group_user,1,0,0,0
-blog_tag_category,blog.tag.category,model_blog_tag_category,website.group_website_designer,1,1,1,1
+blog_tag_category,blog.tag.category,model_blog_tag_category,website.group_website_designer,1,0,0,0
+blog_tag_category_manager,blog.tag.category,model_blog_tag_category,website_blog.group_website_blog_manager,1,1,1,1
+access_website_page_blog_manager,website.page.blog.manager,website.model_website_page,website_blog.group_website_blog_manager,0,0,1,0
+access_website_ir_ui_view_blog_manager,website.ir.ui.view.blog.manager,website.model_ir_ui_view,website_blog.group_website_blog_manager,1,1,1,1

```

Comment:
Do we really need this two access, without giving create access on Website Page and without ir.ui.view access I am able to create blogs.

Reply:
Yes, both access rights are necessary.
The access rights on the Website Page and `ir.ui.view` access are required to enable the sidebar on the `website_blog` page.
Without these, the sidebar won’t appear as expected.